python-sdk/nuscenes/map_expansion/map_api.py

```diff
@@ -476,6 +480,37 @@ def discretize_lanes(self, tokens: List[str],
 
         return {ID: discretize_lane(self.arcline_path_3.get(ID, []), resolution_meters) for ID in tokens}
 
+    def _get_connected_lanes(self, lane_token: str, incoming_outgoing: str) -> List[str]:
+        """
+        Helper for getting the lanes connected to a given lane
+        :param lane_token: Token for the lane.
+        :param incoming_outgoing: Whether to get incoming or outgoing lanes
```

Comment:
Thoughts on using an enum for clarity instead of this `incoming_outgoing` string?